invenio_global_search/config.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 Graz University of Technology.
+#
+# invenio-global-search is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Global Search Configs."""
+
+from invenio_i18n import gettext as _
+
+DUBLIN_CORE_ORIGINAL_SCHEMAS = {
```

Comment:
Move to an instance specific config module, `app-rdm` or `invenio.cfg`.

Reply:
why?